File: Java/BubbleSort.java
import java.util.*;

public class BubbleSort{
    public static void main(String[] args) {
        int[] arr = {4,23,6,1,5,7,8,20};
        System.out.println(Arrays.toString(arr));
        for(int i = 0;i<arr.length-1;i++){
            for(int j = 0;j<arr.length-1-i;j++){
                if(arr[j]>arr[j+1]){
                    int temp = arr[j];
                    arr[j] = arr[j+1];
                    arr[j+1] = temp;
                }
            }
        }
        System.out.println(Arrays.toString(arr));
    }
}

File: Python/BubbleSort.py
def bubbleSort(arr):
    i = 0
    while i<len(arr)-1:
        j=0
        while j<len(arr)-1-i:
            if arr[j]>arr[j+1]:
                tmp = arr[j+1]
                arr[j+1] = arr[j]
                arr[j] = tmp
            j+=1
        i+=1

arr = [4,23,6,1,5,7,8,20]
print(arr)
print()
bubbleSort(arr)
print(arr)


